backend/typescript/rest/entityRoutes.ts

```diff
@@ -0,0 +1,91 @@
+import express, { Router } from 'express';
+import { Entity } from "../models/entity.mgmodel"
+import EntityService from "../services/implementations/EntityService";
+import { EntityRequestDTO } from "../services/interfaces/IEntityService";
+
+export const entityRouter: Router = Router();
+
+/* Create entity Object */
+entityRouter.post("/", async (_req, res) => {
+    const entityObj: EntityRequestDTO = _req.body.entity;
+    const IentityObj = {} as Entity;
+    const entService = new EntityService();
+    IentityObj.stringField = entityObj.stringField;
+    IentityObj.intField = entityObj.intField;
+    IentityObj.enumField = entityObj.enumField;
+    IentityObj.stringArrayField = entityObj.stringArrayField;
+    IentityObj.boolField = entityObj.boolField;
+
+    try {
+        await entService.createEntity(IentityObj);
+        res.send('Success')
```

Comment:
```ts
const entity = await entService.createEntity({
  stringField: req.body.stringField,
  intField: req.body.intField,
  ...,
});
res.status(200).json(entity);
```
I prefer sending the newly created object back (it contains an auto-generated `id` field that the client does not know about), and setting the status code explicitly

Similar changes for the PUT endpoint.